app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'lib/experiments/open-gmail',
+  'lib/experiments/mailcheck',
+  'lib/experiments/coppa-view',
+  'lib/experiments/null',
+  'lib/url'
+], function (OpenGmailExperiment, MailcheckExperiment, CoppaExperiment, NullExperiment, Url) {
+  'use strict';
+
+  var VERIFICATION_TYPE = 'verification';
+  var VERIFICATION_ABLE = 'verificationExperiment';
+  var FORCE_EXPERIMENT_PARAM = 'forceVerificationExperiment';
+  // available verification experiments that must be independent.
+  var VERIFICATION_EXPERIMENTS = {
+    'openGmail': OpenGmailExperiment,
+    'mailcheck': MailcheckExperiment,
+    'coppaView': CoppaExperiment
+  };
+
+  return {
+    /**
+     * Use Able to pick an experiment based on experiment type.
+     * Makes experiment of same independent.
+     *
+     * @private
+     */
+    _initExperimentByType: function (type) {
```

Comment:
We have generally been exposing mixin functions that are supposed to be exposed to the mixed in class as public.